modules/artifact/src/main/java/org/mule/runtime/module/artifact/classloader/RegionClassLoader.java

```diff
@@ -133,4 +148,34 @@ public final URL findResource(final String name) {
 
     return new CompoundEnumeration<>(enumerations.toArray(new Enumeration[0]));
   }
+
+  @Override
+  public void dispose() {
+    for (ArtifactClassLoader classLoader : unfilteredClassLoaders) {
+      try {
+        classLoader.dispose();
+      } catch (Exception e) {
+        // Ignore and continue
```

Comment:
Even thought we can't do anything if there's an error and it should continue, it should log the exception in debug level.